fighthealthinsurance/rest_views.py

```
@@ -559,6 +559,39 @@ def _sanitize(val: str, max_len: int) -> str:
+    """Opt a denial into external models so a re-run of appeal generation
+    can call out to higher-capability cloud LLMs.
+
+    Patients see a button on the appeals page offering this when the
+    initial generation produced few or no appeals from internal models;
+    this endpoint flips `Denial.use_external` so the next generation
+    request includes the external backends. Auth mirrors the appeal
+    stream: the (denial_id, email, semi_sekret) triple is the only
+    credential.
+    """
+
+    permission_classes = [AllowAny]
```

Ruff (0.15.14)

[warning] Mutable default value for class attribute

(RUF012)